### PR TITLE
Optimizations for tuples using. Removed unused types transformations.

### PR DESCRIPTION
**Pull request by umax**
Simple optimizations of tuple using.

### CODE REVIEW

File: prometheus_client/core.py [hunk 1]
```diff
@@ -185,9 +185,8 @@ def __init__(self, name, documentation, value=None, labels=None):
         Metric.__init__(self, name, documentation, 'untyped')
         if labels is not None and value is not None:
             raise ValueError('Can only specify at most one of value and labels.')
-        if labels is None:
-            labels = []
-        self._labelnames = tuple(labels)
+
+        self._labelnames = tuple(labels or [])
```
Comment (brian-brazil) [inline prometheus_client/core.py]: What performance difference does this make?

Comment (umax) [reply prometheus_client/core.py]: No difference, just more shorted and more Pythonyc.
If you prefer, I can remove this changes from PR.

Comment (brian-brazil) [reply prometheus_client/core.py]: It's fine as-is, and I'd expect it to be faster this way

Comment (umax) [reply prometheus_client/core.py]: Updated.

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**